### DISCUSSION TITLE
podcast:social

### DISCUSSION THREAD

**Discussion by picard102** [Podcastindex-org/podcast-namespace#559] [Clarification Needed]
Did this tag get axed? 

The docs seem to indicate it was subsumed by [socialInteract](https://github.com/Podcastindex-org/podcast-namespace/blob/main/proposal-docs/social/social.md), but the two seem be serving different purposes. Social being channel level, and socialInteract being episode specific.

**Comment by dhk2** [comment]
doh, my bad

**Reply by picard102** [reply]
Weird, because socialInteract doc seems pretty explicit it's for comments on an episode. https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#social-interact

**Reply by daveajones** [reply]
It’s meant to be both channel and item eventually. But we started with item only to keep things simple. It will be expanded.

**Reply by picard102** [reply]
When will it be expanded?

**Reply by Dwev** [reply]
Now that we have some excitement around the potential of ActivityPub, should we revisit a `<channel>` enhancement to `<podcast:socialInteract>` or adopting the `<podcast:social>` tag for Phase 7?  Right now, there is no specified way of determining an AP address for a feed.
@daveajones @stevencrader

**Comment by samsethi** [comment]
What's the use case as Channel? Is it just a way to store the social media channels a podcast is using because I think the Contentlink is trying to do the same thing?

**Reply by picard102** [reply]
Contentlink looks to be item specific and is a pseudo alternative enclosure for content of an episode. This would at channel level would be external social links for the entire series rather than episode specific.
